README.md

```diff
@@ -84,14 +84,13 @@ The protocol chooses to override at least the following from https://docs.sherlo
 - Discrepancies in handling of zero/non-actionable input amounts (no-op, revert, etc..) are out of scope.
 - Issues in other Makerdao repositories are out of scope, even in case they are relevant to interactions with the contracts in scope. For example - dss, the Chief contract.
 - As the initiation of the endgame components is not considered time-critical, any permissionless action that can cause the spells to revert is not considered a medium or high severity issue (as another spell can be crafted). It can be assumed that no other mission-critical tasks are included in the same spell.
-- Issues where there is damage to the protocol but attack cost exceeds damage caused/funds stolen significantly are considered low severity.
+- Issues where there is damage to the protocol but attack cost exceeds damage caused/funds stolen significantly (50%+ more) are considered low severity.
```

Comment:
I think "damage caused" and "funds stolen" should be treated differently. The attacker paying `2.5*x` to steal `x` (net cost = `1.5*x`) should be about the same severity as the attacker paying `1.5*x` to grief (but not steal) `x` (net cost = `1.5*x`).
```suggestion
- Issues where there is damage to the protocol/users but the net attack cost exceeds the damage caused significantly (50%+ more) are considered low severity.
```